src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,91 @@
+import podOverviewChart from './img/pod-overview-chart.png'
+import { ImageMeta } from 'posts/post'
+
+const image: ImageMeta = {
+  url: podOverviewChart,
+  width: 1632,
+  height: 502,
+}
+
+const title = 'microCOVID Risk Tracker: How our house got our lives back'
+
+const author = 'Jeremy Blanchard'
+const date = 'February 11, 2021'
+
+const summary =
+  'Navigating house/pod agreements around COVID can be really stressful. We made a tool to help make it a lot easier.'
+const content = `
+
+<style>
```

Comment:
Could we put the css in e.g. Paper.scss instead?

Reply:
Yeah. it's stupid that it's in here. you're right.

Reply:
Done.